tests/test_dagmc_file.py

```diff
@@ -47,9 +47,14 @@ def test_set_native_ranges():
     single_cube = ds.DagmcStats(test_env[1]['input_file'])
     test_pass = np.full(len(single_cube.entity_types), False)
     for i, native_range_type in enumerate(single_cube.entity_types):
-        range = single_cube._my_moab_core.get_entities_by_type(
+        entity_range = single_cube._my_moab_core.get_entities_by_type(
             single_cube.root_set, native_range_type)
-        test_pass[i] = (range == single_cube.native_ranges[native_range_type])
+        if native_range_type == types.MBENTITYSET:
+            # dimension meshsets are of type MBENTITYSE but not in native_ranges
+            num_dim_ms = 4
```

Comment:
I think this should be defined by the size of `self.entityset_ranges`, e.g.
```suggestion
            num_dim_ms = single_cube.entityset_ranges.size()
```